src/views/Publish.vue

```diff
@@ -312,6 +317,11 @@ onMounted(async () => {
     }
     if (root.value) {
       await store.dispatch("fetchProjectConfig");
+      // check if the default keywords are filled in config
+      // updated if the default keywords are not set
+      if (!defaultKeywordsArray.value.every((item) => config.value?.keywords?.includes(item))) {
+        await saveKeywords(defaultKeywordsArray.value);
```

Comment:
We shouldn't save the `defaultKeywordsArray` directly, but to insert what default keyword is missing from the original keywords array to form a new keywords array and save.